packages/market-service/src/coingecko/coingecko.ts

```diff
@@ -1,19 +1,11 @@
 import axios from 'axios'
 import dayjs from 'dayjs'
-import { MarketService, AssetMarketData, HistoryData, HistoryTimeframe } from '../api'
+import { MarketService, MarketData, HistoryData, HistoryTimeframe } from '../api'
+import { ChainTypes } from '@shapeshiftoss/asset-service'
```

Comment:
I think we should move shared types out into a `@shapeshiftoss/types` package

Reply:
Agreed. I want to do that in another PR so I can get this merged before people continue using the old market service for everything and make more changes